resistance-is-futile/usr/share/asteroid-launcher/watchfaces/resistance-is-futile.qml

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (C) 2022 - Ed Beroset <github.com/beroset>
+ * All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.12
+import QtGraphicalEffects 1.12
+
+Item {
+    function getTimeDigit(t, index) {
+        var timestring = t.toLocaleString(Qt.locale(), (use12H.value ? "hhmmss ap" : "HHmmss"));
+        return timestring[index]
+    }
+
+    Rectangle {
+        id: logoArea
+        color: "transparent"
+        anchors {
+            centerIn: parent
+            verticalCenterOffset: -parent.height * 0.272
+        }
+        width: parent.width * 0.12
+        height: parent.height * 0.12
+
+
+        Image {
+            z: 1
+            id: asteroidLogo
+            visible: !displayAmbient
+            source: "../watchfaces-img/asteroid-logo.svg"
+            antialiasing: true
+            anchors.fill: parent
+            opacity: 0.7
+
+            Text {
+                z: 2
+                id: asteroidSlogan
+                visible: !displayAmbient
+                font {
+                    family: "Raleway"
+                    pixelSize: parent.height * 0.31
+                }
+                color: "white"
+                horizontalAlignment: Text.AlignHCenter
+                anchors {
+                    centerIn: parent
+                    verticalCenterOffset: -parent.height * 0.005
+                }
+                text: "<b>AsteroidOS</b><br>Hack Your Wrist"
+            }
+        }
+        MouseArea {
+            anchors.fill: parent
+            onPressAndHold: asteroidLogo.visible ?
```

Comment:
asteroidLogo.visible = !asteroidLogo.visible should work the same and be clearer ?

Reply:
That's on me ;) Will look into also changing the source where this was cloned from.

Reply:
changed